.github/workflows/release.yaml

```diff
@@ -164,3 +157,19 @@ jobs:
         with:
           name: artifacts
           path: dist/**/*
+
+  release-docker-assets:
+    needs: [release]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: Build & Publish docker images
+        run: make release-docker-assets
+        env:
+          GHCR_USERNAME: ${{ github.actor }}
+          GHCR_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
+          DOCKER_USERNAME: ${{ secrets.TOOLBOX_DOCKER_USER }}
```

Comment:
though this step is given the credentials, this wouldn't work until we explicitly run `docker login`. We couldn't do this in a sane way with the signing step (we needed to replace the whole keychain, so the creds would be wiped), but now that there are no signing concerns and this is just a workflow concern (not a step that would ever need to be done locally / in the makefile) then we can use the docker provided action: https://github.com/docker/login-action

Reply:
(same answer for the ghcr credentials)